Helper/Data.php

```diff
@@ -254,6 +254,9 @@ public function getStatusCodes(): array
     public function getTransactionAdditionalInfo(array $array, array $rawInfo = [], string $keyPrefix = ''): array
     {
         foreach ($array as $key => $value) {
+            if (in_array($key, ['brq_websitekey', 'brq_signature', 'brq_payer_hash'])) {
+                continue;
+            }
```

Comment:
what do these things do ?

Reply:
This data shouldn't be displayed on Order Transactions and I removed them from Transaction additional information
![image](https://github.com/buckaroo-it/Magento2/assets/106905746/0dbda91c-32ac-432e-8ff6-f3a701271322)